reflex/event.py

```diff
@@ -704,24 +704,41 @@ def fn():
 
 def redirect(
     path: str | Var[str],
-    external: Optional[bool] = False,
-    replace: Optional[bool] = False,
+    external: Optional[bool] = None,
+    is_external: Optional[bool] = None,
+    replace: bool = False,
```

Comment:
```suggestion
    is_external: Optional[bool] = None,
    replace: bool = False,
    external: Optional[bool] = None,
```

Okay i could nitpick this PR to death i guess? everytime i look at it i see something else to call out; hear me out:

if we move the deprecated parameter to the end of the argument list this has the advantage that previous invocations of this function that were being called with position arguments continue to work fine.

consider

```python
rx.redirect("https://reflex.dev", True, True)
```

Before this PR the arguments would be interpreted as external=True and replace=True; although with the patch as it's currently written the behavior of the function would change, and now it's interpreted as replace=False because of the inserted argument.

If you take the suggestion to rename `external` to `is_external` and move the compat `external` argument _after_ `replace`, then nobody (foolishly) using positional arguments will be broken and there is no behavior change due to the signature changing.

For maximum API stability, a function's semantic signature should _never_ change without good reason.

Reply:
yeah that makes a lot of sense